### ISSUE TITLE
Unity 2017.3 - Unable to Capture video:  WAMDEIA_MUXING_FAILED

### ISSUE THREAD

**Issue by ichabodcole** [facebook/360-Capture-SDK#32]
Thanks for releasing this! While I have been able to take 360 screenshot I have unfortunately not been able to capture video successfully. I'm just loading the included sample project, and attempting to capture video with the surround capture option with a cube map export.
Any help greatly appreciated!

This is the error output from Unity
```
Failed on muxing video and audio data. Please check FBCaptureSDK.log file for more information. [Error Type: WAMDEIA_MUXING_FAILED]
UnityEngine.Debug:Log(Object)
FBCapture.SurroundCapture:MuxingThread() (at Assets/EncodePackage/Script/SurroundCapture.cs:237)
```

This is the contents of the FBCaptureSDK log
```
[ 2018-01-12 00:24:52 ][LOG]: Folder is already existed
[ 2018-01-12 00:24:52 ][LOG]: Added metadata for 360 recognition in FB
[ 2018-01-12 00:24:52 ][LOG]: Succeeded screen capture
[ 2018-01-12 00:24:52 ][LOG]: Screenshot has successfully finished
[ 2018-01-12 00:24:58 ][LOG]: Folder is already existed
[ 2018-01-12 00:24:58 ][LOG]: Video Codec info: NV_ENC_H264
[ 2018-01-12 00:24:58 ][LOG]: Bitrate:: 5000000
[ 2018-01-12 00:24:58 ][LOG]: Device type: DirectX 11
[ 2018-01-12 00:24:58 ][LOG]: Set encode configurations successfully
[ 2018-01-12 00:24:58 ][LOG]: Audio device name: : Headphones (Rift Audio)
[ 2018-01-12 00:24:58 ][LOG]: Found Rift headphone. We're using Rift headphone as audio intput/output source
[ 2018-01-12 00:25:03 ][ERROR]: Muxing failed, error: : 516
```

Unity 2017.3.0f3
Oculus Rift
Oculus 1.22.0
Windows 10
Nividia GTX 1080
Asus X99 Motherboard

-----------------------------------------

OS Name	Microsoft Windows 10 Pro
Version	10.0.16299 Build 16299
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Manufacturer	ASUS
System Model	All Series
System Type	x64-based PC
System SKU	Unsupported
Processor	Intel(R) Core(TM) i7-5820K CPU @ 3.30GHz, 3301 Mhz, 6 Core(s), 12 Logical Processor(s)
BIOS Version/Date	American Megatrends Inc. 3101, 5/4/2016
SMBIOS Version	3.0
BIOS Mode	UEFI
BaseBoard Manufacturer	ASUSTeK COMPUTER INC.

**Comment by cg439** [comment]
This could be related to Windows 10 maybe. We need to do more testing to
confirm, I suspect an audio encoding issue which led to muxing failing. We
should probably in general support zero audio channel muxing so if it
doesn't exist or failed the video just gets muxed into an MP4

**Comment by ichabodcole** [comment]
@cg439 Yah, it would be great if I could at least get video out. Let me know if there is any more info or testing I can provide. Thanks!

**Comment by cg439** [comment]
Do you see any .h264 file in the folder you tried to save the recorded
video to?

**Comment by grancia** [comment]
Current codes in the github are quite outdated(about 6 months). I am supposed to update to latest codes next week. Please check with latest ones.
Thanks

**Comment by ichabodcole** [comment]
@cg439 FYI, I updated the audio driver for my x99-deluxe motherboard, which uses realtek audio, and I can now successfully get video out (YAY!). I have not yet been able to successfully get sound exported into the resulting video, but will keep experimenting and post an update if I am able to solve that.

To answer your question though, when I would start recording I would see two files created in the Gallery folder, which I believe were the video and audio media files, but then both would be deleted as soon as I stopped recording and got the unity console error.

Will check back next week for the latest updates. Thanks for your help (and responsiveness), much appreciated! Feel free to close this ticket as you see fit.
